config/winston.ts

```diff
@@ -0,0 +1,88 @@
+import * as winston from "winston";
```

Comment:
it's not configuration. 
It's type so better to choose the relevant naming convention.